tucson-teststand/MT/docker-compose.yaml

```diff
@@ -0,0 +1,52 @@
+version: "3.6"
+
+services:
+
+  mtm1m3-sim:
+    image: ts-dockerhub.lsst.org/mtm1m3_sim:${CYCLE}
+    container_name: mtm1m3-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-mtm1m3.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-mtm1m3.log
+    volumes:
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "5"
+        max-size: "100m"
+
+  mtm2-sim:
+    image: ts-dockerhub.lsst.org/m2:${CYCLE}
+    container_name: mtm2-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-mtm2.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-mtm2.log
+    volumes:
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "5"
+        max-size: "10m"
+
+networks:
+  default:
+    external:
+      name: test-stand-network
```

Comment:
I don't think any container is using this network. I imagine you can probably remove it.

Reply:
Done.